### PR TITLE
TEST: stats unit tests for mann-whitney-u and wilcoxon-srt

### PR DESCRIPTION
**Pull request by lizgehret**

### CODE REVIEW

**Comment by lizgehret** [comment]
[Jupyter Notebook link to manual mann-whitney testing](https://gist.github.com/lizgehret/0200c5ed7f49c91c5b25c3905cae6661)

**Comment by lizgehret** [comment]
[Jupyter Notebook](https://gist.github.com/lizgehret/c9add7b451e5e91b1017a2a963276bff) with manual testing for mann-whitney-u calculations

File: q2_fmt/tests/test_engraftment.py [hunk 1]
```diff
@@ -476,13 +483,96 @@ def test_wilcoxon_with_faith_pd_consecutive_asymptotic(self):
...
+                                    " within the distribution."):
+            wilcoxon_srt(distribution=self.faithpd_timedist,
+                         hypothesis='baseline', baseline_group='foo')
+
     # Mann-Whitney U test cases
-    # search for any online refs for datasets with given results to compare
+
+    # Data in the exp_stats_data dataframes were calculated 'by hand' in a jupyter notebook
+    # using the same data, manually organized into groups and subsequently compared using
+    # scipy.stats.mannwhitneyu to calculate the test-statistic and p-values
```
Comment (ebolyen) [inline q2_fmt/tests/test_engraftment.py:510]: I would throw the link to your gist here as well.

**Review by ebolyen — APPROVED** [review]
Tiny comment, lgtm otherwise!

**Pull request merged**